components/input/input.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="../../assets/Wings.svg" />
+    <link rel="stylesheet" type="text/css" href="../../utilities/doc.css" />
+    <title>Wings UI | Input</title>
+  </head>
+  <body>
+    <header id="home-header" class="flex-row flex-space-btw">
+      <h1 class="home-header-title"><a href="/">WINGS UI</a></h1>
+      <nav class="home-nav flex-row">
+        <ul class="home-nav-list ls-none">
+          <li class="home-nav-item">
+            <a class="home-nav-link" href="/">Home</a>
+          </li>
+          <li class="home-nav-item">
+            <a class="home-nav-link nav-link-active" href="../alert/alert.html"
+              >Docs</a
+            >
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main id="doc-main" class="flex-row">
+      <aside class="doc-sidebar">
+        <ul class="sidebar-list ls-none mp-m">
+          <strong class="sidebar-header font-m">Components</strong>
+          <li class="sidebar-item mp-m">
+            <a href="../alert/alert.html" class="sidebar-link">Alert</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../avatar/avatar.html" class="sidebar-link">Avatar</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../badge/badge.html" class="sidebar-link">Badge</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../button/button.html" class="sidebar-link">Button</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../card/card.html" class="sidebar-link">Card</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../image/image.html" class="sidebar-link">Image</a>
+          </li>
+          <li
+            class="sidebar-item mp-m active-component"
+            style="background-color: var(--input-color)"
+          >
+            <a href="./input.html" class="sidebar-link">Input</a>
+          </li>
+        </ul>
+      </aside>
+      <section class="doc-section mp-m">
+        <h2 class="doc-header font-xl">Input</h2>
+        <p class="doc-text mpy-s">
+          Wing UI’s has simple input form and form fields with validation. These
+          form controls are easy to style using existing utilities.
+        </p>
+
+        <!-- || Input Form-->
+        <div
+          class="component-type mp-s"
+          style="border-color: var(--input-color)"
+        >
+          <h3 class="component-header font-l">Input Form</h3>
+          <p class="component-text mpy-s">
+            This is a simple input form to layout in order to get user reviews/
+            feedback /comments, etc.
+          </p>
+          <div class="component-display mp-m">
+            <form class="input-form">
```

Comment:
You can name this class name as form-container or something, because input can be confused with field in CSS.

Reply:
Yes will change it! Thanks @shashidwivedi